core/domain/exp_services.py

```diff
@@ -1622,6 +1623,37 @@ def get_next_page_of_all_non_private_commits(
     ) for entry in results], new_urlsafe_start_cursor, more)
 
 
+def get_image_filenames_from_exploration(exploration):
+    """Get the image filenames from the exploration.
+
+    Args:
+        exploration: Exploration object. The exploration itself.
+
+    Returns:
+       list(str). List containing the name of the image files in exploration.
+    """
+    filenames = []
+    for state in exploration.states.itervalues():
+        if state.interaction.id == 'ImageClickInput':
+            filenames.append(state.interaction.customization_args[
+                'imageAndRegions']['value']['imagePath'])
+
+    html_list = exploration.get_all_html_content_strings()
+    rte_components_in_exp = []
+    for html_string in html_list:
+        rte_components_in_exp = (
+            rte_components_in_exp + html_cleaner.get_rte_components(
+                html_string))
+
+    for rte_comp in rte_components_in_exp:
+        if 'id' in rte_comp and (
+                str(rte_comp['id']) == 'oppia-noninteractive-image'):
+            filenames.append(rte_comp['customization_args'][
+                'filepath-with-value'])
```

Comment:
You can add the linebreak after the `append(`

Reply:
Done!